src/components/global-toolbar/global-toolbar.scss

```diff
@@ -51,3 +51,13 @@
 .pipeline-global-routes-toolbar a.active .pipeline-menu-button--link svg {
   opacity: 1;
 }
+
+.update-reminder-dot {
+  background-color: $color-accent-yellow;
+  border-radius: 50%;
+  height: 8px;
+  position: absolute;
+  right: 28px;
+  top: 31px;
+  width: 8px;
+}
```

Comment:
'dot' on the SettingsIcon is not positioned properly, here is the screenshot how it looks on my Mac display:

<img width="565" alt="Screenshot 2022-07-05 at 13 58 32" src="https://user-images.githubusercontent.com/88193161/177324691-fcc9e648-0289-43a4-8883-c23a3c531eef.png">


I think `svg` and `span` elements need to be both positioned inside the same element, to be properly shown on different screen sizes.

Reply:
Can you kill your frontend server and then restart it please? I have a feeling the code needs to compile for you to see it correctly.

Reply:
Yes, it's fine now :D